src/components/Question.js

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+const Question = ({ question, answers, correct, id, checked, setSuccess }) => {
+  const [submitted, setSubmitted] = useState(null);
+
+  const responseElement = answers.map((response, index) => (
+    <div
+      key={index}
```

Comment:
I wouldn’t recommend using indexes for keys when mapping, if the order of items may change. This can negatively impact performance and may cause issues with the component state. To understand and know more about why this is not a best practice you can read this [article](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/).

Reply:
okay, since I don't have an id property to use, and I don't want to install the id generator package for now. However, I have understood that it's a bad pattern... next time I will try to avoid using it. and thanks for the article 😊

Reply:
Sure